crates/networking/p2p/src/network.rs

```diff
@@ -253,6 +286,50 @@ impl Network {
             }
         }
     }
+
+    fn handle_gossipsub_event(&mut self, event: gossipsub::Event) {
+        info!("Gossipsub event: {:?}", event);
+        match event {
+            gossipsub::Event::Message {
```

Comment:
```suggestion
            GossipsubEvent::Message {
```

could we just import `Event as GossipsubEvent`